README.md

```diff
@@ -21,34 +21,19 @@ finding something going wrong. There's a long list of known issues
 Check out [the wiki page](https://github.com/new-xkit/XKit/wiki/Writing-a-New-Extension)!
 
 ## Develop XKit
-Download [Node.js](https://nodejs.org/download/) for your platform.
+Review the project prerequisites and learn how to build XKit from source [over on the wiki](/wiki/Build-XKit).
```

Comment:
I don't think it's safe to assume that each fork has a wiki set up. For example, while you have [yours](https://github.com/0xazure/XKit/wiki) set up, [I](https://github.com/bvtsang/XKit/wiki) and [blackjackkent](https://github.com/blackjackkent/XKit/wiki) don't.

For this case, an absolute URL might be better.